bigframes/dataframe.py

```diff
@@ -3674,7 +3674,11 @@ def apply(self, func, *, axis=0, args: typing.Tuple = (), **kwargs):
             return result_series
 
         # Per-column apply
-        results = {name: func(col, *args, **kwargs) for name, col in self.items()}
+        if hasattr(func, "bigframes_remote_function"):
+            results = {name: col.apply(func) for name, col in self.items()}
+        else:
+            results = {name: func(col, *args, **kwargs) for name, col in self.items()}
```

Comment:
I believe this already has test coverage, let's double check

Reply:
Yeah, we have several tests for it already. Confirmed!